ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -30,9 +30,17 @@ sub valid_sshd_file
 {
     my ($self, $file, $cfg) = @_;
 
+    my $sshd_bin = $cfg->{sshd_path} || DEFAULT_SSHD_PATH;
+    my $validate_optional = $cfg->{validate_optional};
+    
+    if (defined($validate_optional) && $validate_optional && ! -x $sshd_bin) {
```

Comment:
there'no need for the `defined()` check (and than neither for the new variable). `if ($cfg->{validate_optional} && ...)` does the same thing